common/test/acceptance/tests/studio/test_studio_settings_details.py

```diff
@@ -194,3 +194,46 @@ def test_entrance_exam_has_unit_button(self):
             css_selector='.add-item a.button-new',
             text='New Subsection'
         ))
+
+
+class SettingsEnrollmentEditTest(StudioCourseTest):
+    """
+    Tests for enrollment end edit permissions in Studio's settings tab
+    """
+    def setUp(self, is_staff=False):
+        super(SettingsEnrollmentEditTest, self).setUp(is_staff=is_staff)
+        self.settings_detail = SettingsPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+        # Before every test, make sure to visit the page first
+        self.settings_detail.visit()
+        self.assertTrue(self.settings_detail.is_browser_on_page())
+
+    def test_enrollment_end_date_edit_permissions_non_global(self):
+        """
+        Test that non-global staff sees messaging that editing the enrollment end date is disabled.
+        """
+        self.settings_detail.visit()
+
+        self.assertTrue(element_has_text(
```

Comment:
Someplace, either here or in the unit test, I would like the editability tested (vs. only the message).